Tools/Source/libs/AOERandomizer.Model/Enums/TeamConfig.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace AOERandomizer.Model.Enums
+{
+    public enum TeamConfig
```

Comment:
Looks good, optional if we can load from config but I am assuming that team sizes won't be changing in AoE4

Reply:
this object is saved to the config, lel.  It binds to the dropdown in the teams page, and remembers the decision.  If team sizes in AoE ever increase their max size to > 8 players, just need to update this enum and everything else UI-wise should be handled.  Would just have to handle the new configurations when spinning, etc.

Reply:
HAVE TO UPDATE HMM THAT MEANS A NEW BUILD